autoextract/aio/client.py

```diff
@@ -179,7 +177,7 @@ async def request_raw(query: Query,
         agg_stats = AggStats()  # dummy stats, to simplify code
 
     if max_query_error_retries and not handle_retries:
-        logger.warning(
+        warnings.warn(
```

Comment:
stacklevel=2 could make this warning more useful, as it'd be pointing to the caller code, not inside the function